CONTRIBUTING.md

```diff
@@ -0,0 +1,148 @@
+
+A PR is considered **ready to merge** when:
+
+* It has received two qualified approvals[^1].
+
+	This is not enforced through automation, but needs to be validated by the
+	maintainer merging.
+	* PRs introducing changes that have already been discussed and consensus
+		reached only need one qualified approval. The discussion and resolution
+		needs to be linked to the PR.
+	* Trivial changes[^2] only need one qualified approval.
+
+* All feedback has been addressed.
+	* All PR comments and suggestions are resolved.
+	* All GitHub Pull Request reviews with a status of "Request changes" have
+		been addressed. Another review by the objecting reviewer with a different
+		status can be submitted to clear the original review, or the review can be
+		dismissed by a [Maintainer] when the issues from the original review have
+		been addressed.
+	* Any comments or reviews that cannot be resolved between the PR author and
+		reviewers can be submitted to the community [Approver]s and [Maintainer]s
+		during the weekly SIG meeting. If consensus is reached among the
+		[Approver]s and [Maintainer]s during the SIG meeting the objections to the
+		PR may be dismissed or resolved or the PR closed by a [Maintainer].
+	* Any substantive changes to the PR require existing Approval reviews be
+		cleared unless the approver explicitly states that their approval persists
+		across changes. This includes changes resulting from other feedback.
+		[Approver]s and [Maintainer]s can help in clearing reviews and they should
+		be consulted if there are any questions.
+
+* The PR branch is up to date with the base branch it is merging into.
+	* To ensure this does not block the PR, it should be configured to allow
+		maintainers to update it.
+
+* It has been open for review for at least one working day. This gives people
+	reasonable time to review.
+	* Trivial changes[^2] do not have to wait for one day and may be merged with
+		a single [Maintainer]'s approval.
+
+* All required GitHub workflows have succeeded.
+* Urgent fix can take exception as long as it has been actively communicated
+	among [Maintainer]s.
+
+Any [Maintainer] can merge the PR once the above criteria have been met.
+
+[^1]: A qualified approval is a GitHub Pull Request review with "Approve"
+	status from an OpenTelemetry Profiler [Approver] or [Maintainer].
+[^2]: Trivial changes include: typo corrections, cosmetic non-substantive
+	changes, documentation corrections or updates, dependency updates, etc.
+
+## Approvers and Maintainers
+
+### Approvers
+
+- [Christos Kalkanis](https://github.com/christos68k), Elastic
+- [Florian Lehner](https://github.com/florianl), Elastic
+- [Tim Rühsen](https://github.com/rockdaboot), Elastic
+
+### Maintainers
+
+- [Dmitry Filimonov](https://github.com/petethepig), Pyroscope/Grafana-
+- [Felix Geisendörfer](https://github.com/felixge), Datadog
+- [Joel Höner ](https://github.com/athre0z), Elastic
+- [Timo Teräs](https://github.com/fabled)
```

Comment:
Things change and so do memberships of such groups. To keep it more generic, https://github.com/orgs/open-telemetry/teams/ebpf-profiler-approvers and https://github.com/orgs/open-telemetry/teams/ebpf-profiler-maintainers should be mentioned here instead of personal accounts.
```suggestion
@open-telemetry/ebpf-profiler-approvers and @open-telemetry/ebpf-profiler-maintainers 
```

Reply:
While I'm happy to mention the groups and not the folks here, every other SIG does it like this.
And updating this list is part of the maintainer onboarding tasks.

https://github.com/open-telemetry/community/pull/1924/files#diff-2babc18ce7c6fdc68c4161df08fb505a258f0bce4dda17f081d7628b0afa04c7R64

Reply:
Fair point - I wasn't aware of this additional manual step in this process.